.github/workflows/ci.yml

```diff
@@ -37,73 +37,10 @@ jobs:
             cd ${GITHUB_WORKSPACE}/build
             cmake .. -DBUILD_SKETCHES=On
             make -j 4
-      - name: Test parser
+      - name: Run CTest
```

Comment:
I think I'd keep the original ci workflow. Good to have this feature, but on github action, I'd like to see individual test results separately.

Reply:
That's fine, I just dropped that commit.